akka-persistence-typed/src/main/scala/akka/persistence/typed/internal/EventsourcedStashManagement.scala

```diff
@@ -0,0 +1,76 @@
+package akka.persistence.typed.internal
+
+import java.util.Locale
+
+import akka.actor.typed.{ ActorSystem, Behavior }
+import akka.actor.{ DeadLetter, StashOverflowException }
+import akka.actor.typed.scaladsl.{ ActorContext, StashBuffer }
+import akka.annotation.InternalApi
+import akka.event.Logging.LogLevel
+import akka.event.{ Logging, LoggingAdapter }
+import akka.persistence._
+import akka.util.ConstantFun
+import akka.{ actor ⇒ a }
+
+/** INTERNAL API: Stash management for persistent behaviors */
+@InternalApi
+private[akka] trait EventsourcedStashManagement {
+  import EventsourcedStashManagement._
+  import akka.actor.typed.scaladsl.adapter._
+
+  protected def log: LoggingAdapter
+
+  protected def extension: Persistence
+  protected def context: ActorContext[Any]
+
+  protected val internalStash: StashBuffer[Any]
+
+  private lazy val logLevel = {
+    val configuredLevel = extension.system.settings.config
+      .getString("akka.persistence.typed.log-stashing")
+    Logging.levelFor(configuredLevel).getOrElse(OffLevel) // this is OffLevel
```

Comment:
Maybe default to 'warn'?

Reply:
those stashes are very expected (while you're recovering or persisting they happen all the time), normally you don't want to see them.

Reply:
aah yes I misread - makes sense

Reply:
What does the comment mean "// this is OffLevel"?